src/components/DropdownContent/index.tsx

```diff
@@ -12,13 +14,27 @@ function createEnum<T extends { [P in keyof T]: P }>(o: T) {
   return o
 }
 
-const DropdownContent: FC<IDropdownContent> = ({ propertySet }) => (
-  <ul className="DropdownContent">
-    {Object.entries<TPropertiesValues>(propertySet).map(
-      ([propertyKey, propertyValue]: [string, TPropertiesValues], idx: number) => (
-        <DropdownItem key={idx} {...{ propertyKey, propertyValue }} />
-      ))}
-  </ul>
-);
+const DropdownContent: FC<IDropdownContent> = ({ propertySet }) => {
+  const [isCopied, setIsCopied] = useState(false);
+
+  const onListClick = useCallback(() => {
+    navigator.clipboard.writeText(JSON.stringify(propertySet));
```

Comment:
To save an indent formatting. 
```suggestion
    navigator.clipboard.writeText(JSON.stringify(propertySet, null, "\t"));
```